### ISSUE TITLE
Help wanted - testing backupheim for cloud save compatibility.

### ISSUE THREAD

**Issue by lauren-mods** [lauren-mods/Valheim-modding#2]
Hello, I have updated Backupheim so that it should offer backups of cloud saves. However, I am not sure if it works with the default steam install (c drive) as mine is installed on the D drive.

It would be awesome if someone who has a Program Files x(86) install could test out the new file [here](https://github.com/lauren-mods/Valheim-modding/blob/main/Backupheim/backupheim-beta.bat)

You would need to input your steam id in the section: 
```
rem Steam ID setup. Replace 'here' with your steam id.

set "steamid=none"
```
which can be found in steam> userdata> (it is the number of the next folder).

Basically I need to check if x copy works with this install location due to the spaces in the 'program files x(86) folder name.

Thanks!

**Comment by lauren-mods** [comment]
[backupheim-beta.zip](https://github.com/lauren-mods/Valheim-modding/files/9127621/backupheim-beta.zip)
Here's the file for easer download.

**Comment by lauren-mods** [comment]
Potentially fixed using ZeroFoxGiven's method (~1 for spaces).
This is beta-3:
[backupheim.zip](https://github.com/lauren-mods/Valheim-modding/files/9180634/backupheim.zip)
